docs/sources/clients/promtail/configuration.md

```diff
@@ -353,6 +353,9 @@ job_name: <string>
 # Configuration describing how to pull/receive Google Cloud Platform (GCP) logs.
 [gcplog: <gcplog_config>]
 
+# Configuration describing how to get Azure Event Hubs messages.
```

Comment:
Is it Azure Event Hub (singular) or Azure Event Hubs (plural)?

Reply:
Great question 👍 

You are right. Its naming is inconsistent in this PR.

The service is called Event Hubs (plural), and I renamed it everywhere now to use "hubs" instead of "hub". 

I added a link to the Azure catalog in this comment https://github.com/grafana/loki/pull/8787#discussion_r1145895076